packages/core/src/components/map-base-layers/map-base-layers.tsx

```diff
@@ -18,7 +21,21 @@ export class MapBaseLayers {
     this.baseLayerChange.emit({ activeBaseLayer: baseLayer });
   }
 
+  componentDidRender() {
+    this.baseLayers
+      .filter(l => !l.disabled && this.previousBaselayers?.find(p => p.id === l.id)?.disabled === true)
+      .forEach(o => {
+        this.selectableRefs[o.id]?.toggleInfo(false);
```

Comment:
Dus, dit zorgt ervoor dat als het `MapBaseLayers` component (opnieuw) rendert, altijd die info op "UIT" getoggled wordt, right?

Reply:
Klopt. `toggleInfo()` is een instance method op het Selectable component. Omdat de state van het info component in Selectable wordt beheerd (en ik het daar wil houden) is er geen fraaie JSX/declaratieve notatie mogelijk.